charms/knative-operator/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: knative-operator
+summary: Knative operator draft
+description: |
+  DRAFT
+containers:
+  knative-operator:
+    resource: knative-operator-image
+resources:
+  knative-operator-image:
+    type: oci-image
+    description: OCI image for knative-operator
+    upstream-source: gcr.io/knative-releases/knative.dev/operator/cmd/operator@sha256:a8f99f8adba8498942d7600b42aea6c7b06522631b213414185fc0126ac4f71b
```

Comment:
+1

Reply:
I've found a registry for all knative images. They have tags that point to specific versions, so it can be something like `gcr.io/knative-releases/knative.dev/operator/cmd/operator:v1.1.0` [Here's](https://console.cloud.google.com/gcr/images/knative-releases/global/knative.dev/operator/cmd/operator) the repo for operator.